targets/TARGET_STM/TARGET_STM32WB/i2c_device.h

```diff
@@ -72,11 +72,11 @@ static inline uint32_t get_i2c_timing(int hz)
     }
     return tim;
 }
+:
```

Comment:
oups...

Reply:
once updated let us know, to restart tests